packages/flutter_tools/lib/src/cache.dart

```diff
@@ -112,7 +112,17 @@ class Cache {
 
   String _dartSdkVersion;
 
-  String get dartSdkVersion => _dartSdkVersion ??= platform.version;
+  String get dartSdkVersion {
+    if (_dartSdkVersion == null) {
+      // Make the version string more customer-friendly.
```

Comment:
Might help for future reference if we add a comment about what the current output looks like and what we're expecting to change it to?

e.g. 
```
// Changes 'Dart 2.1.0-dev.8.0.flutter-4312ae32' to 'Dart-2.1.0-dev8.0'
```

Reply:
Done.